From fd1b7e06541dd9c6e14620f9b8cc23773315ea2f Mon Sep 17 00:00:00 2001
From: "Yang, Dong" <dong.yang@intel.com>
Date: Thu, 16 Apr 2015 15:05:44 +0800
Subject: [PATCH 386/462] adjust the resolution to remove the green color line

there is a color line at the edge of the preview window
this is because the sensor resolution not configed properly
adjust the reolution to remove the color line

Change-Id: I6802b3f3a1fb5f69a0aee80883086e14818507c9
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-28890
Signed-off-by: Yang, Dong <dong.yang@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/54291
Build: IMIN (SC)
Approver: Shuguang Gong <shuguang.gong@intel.com>
Reviewed-by: Shuguang Gong <shuguang.gong@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/gc2235.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/gc2235.h b/drivers/external_drivers/camera/drivers/media/i2c/gc2235.h
index 6a71120..66e9b25 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/gc2235.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/gc2235.h
@@ -441,7 +441,7 @@ static struct gc2235_reg const gc2235_1616_1082_30fps[] = {
 	{ GC2235_8BIT, 0x0d, 0x04 }, /* win height 1232 */
 	{ GC2235_8BIT, 0x0e, 0xd0 },
 	{ GC2235_8BIT, 0x0f, 0x06 }, /* win width: 1616 */
-	{ GC2235_8BIT, 0x10, 0x60 },
+	{ GC2235_8BIT, 0x10, 0x50 },
 
 	{ GC2235_8BIT, 0x90, 0x01 },
 	{ GC2235_8BIT, 0x92, 0x4a },
@@ -482,7 +482,7 @@ static struct gc2235_reg const gc2235_1616_1216_30fps[] = {
 	{ GC2235_8BIT, 0x0d, 0x04 }, /* win height 1232 */
 	{ GC2235_8BIT, 0x0e, 0xd0 },
 	{ GC2235_8BIT, 0x0f, 0x06 }, /* win width: 1616 */
-	{ GC2235_8BIT, 0x10, 0x60 },
+	{ GC2235_8BIT, 0x10, 0x50 },
 
 	{ GC2235_8BIT, 0x90, 0x01 },
 	{ GC2235_8BIT, 0x92, 0x02 },
-- 
1.9.1

